docs/cloud/cheatsheet.md

```diff
@@ -51,125 +83,108 @@ sudo ./edit-config netdata.conf
 ```
 
-```
+```conf
 [plugins]
  go.d = yes # enabled
  node.d = no # disabled
 ```
 
 #### Enable/disable specific collectors
 
+```bash
+sudo ./edit-config go.d.conf # edit a plugin's config
 ```
-sudo ./edit-config go.d.conf
-```
-
-> `Or python.d.conf, node.d.conf, edbpf.conf, and so on`.
 
-```
+```conf
 modules:
  activemq: no # disabled
- bind: no # disabled
  cockroachdb: yes # enabled
 ```
 
-#### Edit a collector's config (example)
+#### Edit a collector's config
 
+```bash
+sudo ./edit-config go.d/mysql.conf
 ```
-$ sudo ./edit-config go.d/mysql.conf
-$ sudo ./edit-config ebpf.conf
-$ sudo ./edit-config python.d/anomalies.conf
-```
-
-## Configuration
-
-#### The Netdata config directory: `/etc/netdata`
 
-> If you don't have such a directory:
-> 📄 [Find your netdata.conf file](https://github.com/netdata/netdata/blob/master/daemon/config/README.md)
-> The cheatsheet assumes you’re running all commands from within the Netdata config directory!
+### Alarms & notifications
 
-#### Edit Netdata's main config file: `$ sudo ./edit-config netdata.conf`
-
-#### Edit Netdata's other config files (examples):
-
-- `$ sudo ./edit-config apps_groups.conf`
-- `$ sudo ./edit-config ebpf.conf`
-- `$ sudo ./edit-config health.d/load.conf`
-- `$ sudo ./edit-config go.d/prometheus.conf`
-
-#### View the running Netdata configuration: `http://NODE:19999/netdata.conf`
-
-> Replace `NODE` with the IP address or hostname of your node. Often `localhost`.
-
-## Alarms & notifications
-
-#### Add a new alarm
+<!-- #### Add a new alarm
 
 ```
 sudo touch health.d/example-alarm.conf
 sudo ./edit-config health.d/example-alarm.conf
+``` -->
+After any change, reload the Netdata health configuration:
+
+```bash
+netdatacli reload-health
+#or if that command doesn't work on your installation, use:
+killall -USR2 netdata
 ```
 
 #### Configure a specific alarm
 
-```
+```bash
 sudo ./edit-config health.d/example-alarm.conf
 ```
 
 #### Silence a specific alarm
 
-```
+```bash
 sudo ./edit-config health.d/example-alarm.conf
  to: silent
 ```
 
-#### Disable alarms and notifications
+<!-- #### Disable alarms and notifications
 
-```
+```conf
 [health]
  enabled = no
-```
-
-> After any change, reload the Netdata health configuration
-
-```
-netdatacli reload-health
-```
-
-or if that command doesn't work on your installation, use:
+``` -->
 
-```
-killall -USR2 netdata
-```
+---
 
-## Manage the daemon
+### Manage the daemon
 
 | Intent                      |                                                                Action |
 | :-------------------------- | --------------------------------------------------------------------: |
 | Start Netdata               |                                      `$ sudo systemctl start netdata` |
 | Stop Netdata                |                                       `$ sudo systemctl stop netdata` |
 | Restart Netdata             |                                    `$ sudo systemctl restart netdata` |
```

Comment:
These should use the `service` command, which should work on almost all platforms (essentially everything that we support except for macOS and some systems that use `runit`), instead of `systemctl`, which will only work on some platforms. Syntax is exactly the same, the only difference is the command itself.

Reply:
changed it to `service`